L1TriggerConfig/Utilities/src/L1TCaloParamsViewer.cc

```diff
@@ -218,13 +230,13 @@ void L1TCaloParamsViewer::analyze(const edm::Event& iEvent, const edm::EventSetu
 
     if( !ptr1->tauCalibrationLUT()->empty() ){
         cout<<"  tauCalibrationLUT=      ["<<ptr1->tauCalibrationLUT()->maxSize()<<"] "<<flush;
-        int tauCalibration[ptr1->tauCalibrationLUT()->maxSize()];
-        for(unsigned int i=0; i<ptr1->tauCalibrationLUT()->maxSize(); i++)
+        int tauCalibration[512]; //ptr1->tauCalibrationLUT()->maxSize()];
```

Comment:
LUTs actually do have "size" equivalent (maxSize in this case). The problem with LUTs is that I calculate hash sums to know if they are identical or not between different payloads. This allows me to get away without printing and diffing their content. What I've discovered is that a sort of paddings shows up when comparing online with offline payloads. In that case experts claim that LUTs are "physically" equivalent. But hash sum of, for example, an array of 256 "0" elements is different from the 512 zeros. So I've hardcoded a the "relevant" size here.